dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1351,6 +1351,11 @@ public ProgramOutput startServerWithArgs(boolean preClean, boolean cleanStart,
             addJava2SecurityPropertiesToBootstrapFile(f, GLOBAL_DEBUG_JAVA2SECURITY);
             String reason = GLOBAL_JAVA2SECURITY ? "GLOBAL_JAVA2SECURITY" : "GLOBAL_DEBUG_JAVA2SECURITY";
             Log.info(c, "startServerWithArgs", "Java 2 Security enabled for server " + getServerName() + " because " + reason + "=true");
+
+            // If we are running on Java 18+, then we need to explicitly enable the security manager
+            if (info != null && info.majorVersion() >= 18) {
```

Comment:
info isn't null above so it can't be null here.

Reply:
noted and updated.